src/util/address.rs

```diff
@@ -1196,7 +1196,8 @@ mod tests {
     fn p2tr_from_untweaked(){
         //Test case from BIP-086
         let internal_key = schnorrsig::PublicKey::from_str("cc8a4bc64d897bddc5fbc2f670f7a8ba0b386779106cf1223c6fc5d7cd6fc115").unwrap();
-        let address = Address::p2tr(Secp256k1::new(), internal_key,None, Network::Bitcoin);
+        let secp = Secp256k1::verification_only();
+        let address = Address::p2tr(&secp, internal_key,None, Network::Bitcoin);
```

Comment:
If there's another round of changes, there could be space added before `None` but please don't bother with it otherwise.